### ISSUE TITLE
Невозможно начать игру или создать сервер если в liblist.gam/gameinfo.txt путь к серверной библиотеке указан к другой папке мода

### ISSUE THREAD

**Issue by Vladislav4KZ** [FWGS/xash3d-fwgs#572]
Например в Dark Future путь к gamedll указан как  "../gearbox/dlls/opfor.dll"
Opposing Force у меня установлен, но на моде который использует серверку из опфора невозможно играть, только если перенести файл в папку dlls самого мода и изменить путь в gameinfo.txt

**Comment by Vladislav4KZ** [comment]
Half-Life Operation Rosenberg, тоже использует dll'ки от Opposing Force

**Comment by Vladislav4KZ** [comment]
И ещё заметил, если папка gearbox указана как fallback_dir, и путь к gamedll указывает к папке dlls/opfor.dll не выходя на родительскую папку, а в самом моде нет папки dlls то ксаш берёт этот opfor.dll с папки gearbox/dlls

**Comment by mittorn** [comment]
А клиентка при этом какая используется? У нас файловая система наследуется от fallback, соответственно если библиотеки нет то будет использовать от фоллбэка. А выйти в parent от gamedir ФС просто не даёт, возможно надо разрешить как-то это отдельно

**Comment by a1batross** [comment]
Да, "../" скорее напарывается на CheckNastyPath. Тем более это все равно чушь какая-то. 

А ФС наследуется так: basedir -> gamedir. Если в геймдире он не нашел дллку, будет в basedir. С каждым модом уровня "набор карт" это всегда работало.
